concepts/00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,158 @@
+
+## Drawbacks
+
+- The team will need to attend a few more meetings. The daily standups, as currently designed, lose their focus on individual areas of the project per day; this was useful in case an external contributor was interested in one specific area of the project. However, we hope the added free-form discussion time will allow for relevant questions to be asked and talked about anyway.
+
+- Implementing some of the automation exactly as specified may end up being a challenge. There's further discussion of this below.
+
+- Bisecting our backlog into different projects has its benefits, but it also precludes a nice, unified view of what needs to be done on the project. This would be addressable with a `backlog` label if we considered that a large-enough problem.
+
+## Rationale and alternatives
+
+There are many ways to approach sprints, and we can't claim this is the best possible sprint methodology out there; only time will tell if this works for us once we start using it. Additionally, this process is not set in stone; anything can change from one sprint to the next. However, there are a few nice characteristics worth highlighting:
+
+- We are still a fairly small team, and we like some flexibility in our process. The modular design of our boards supports that -- for example, if we consider a new issue or pull request particularly important, we can manually add it directly to our Active Sprint board and have it skip the backlog entirely.
+- Organizing our backlog with boards of their own should help us better digest our backlog as a whole, and will support modularization of the monolithic Enarx repository down the line.
+- Having a single "Active Sprint" board allows visitors to see a snapshot of what's currently a top priority for Enarx.
+- With automation, this process has potential to minimize the busywork that project developers have to deal with, and only require human judgment when it is needed.
+- Running through a triage every day ensures issues are promptly reviewed by the team after they are submitted.
+
+## Prior art
+
+Our current process of topic-specific boards largely informs this proposal's approach to the backlog. In fact, this proposal could be considered an extension or adaptation of our current process in some ways.
+
+And, of course, the agile sprint methodology is widely employed just about everywhere in software; see [Atlassian](https://www.atlassian.com/agile/scrum/sprints) and [Trello](https://blog.trello.com/beginners-guide-scrum-and-agile-project-management) for good examples. And, of course, Red Hat [itself](https://developers.redhat.com/blog/2017/06/29/agile-software-development-the-red-hat-way/) uses it.
+
+## Unresolved questions
+
+The most significant unresolved question does not involve process, but rather the automation we hope to pair with it: whether the Active Sprint board will live inside the main Enarx repository, or as an all-encompassing Enarx organization board.
```

Comment:
Disagree: for me, the most signficant unresolved question revolves around role definitions and the responsibility of each role in the various steps in the process.

Reply:
Automation was my first flag early on and I see it addressed later in the text. I appreciate the breakdown of concepts in the document. Please forgive my ignorance, I must ask how much this alternate process will streamline efforts. More to the point, does this process reduce or increase overall administrative overhead in addition to engineer related tasks? If it does streamline, is there an associated risk of "paralysis by analysis?"

The document indicates a notable level of uncertainty regarding the automation of this process and a less centralized use of a platform. Is there an understood exit strategy if the transition fails (eg automation) and the project needs to revert to existing project management methods, in whole or in part?